test/cwl/PZEstimationPipe.cwl

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env cwl-runner
+
+arguments:
+- {position: -1, valueFrom: -mceci_example.example_stages}
```

Comment:
Not sure where this is being generated but I've found it's much much safer to always just call it with -m ceci_example, not including the full dotted path.  The latter means the subpackage is executed before the parent package which inevitably causes the double import issue that you've seen.

Reply:
ohhhh that's why I was having issues... Ok, I'll fix that, thanks !